build.gradle

```diff
@@ -634,6 +634,7 @@ ext.createDiffFile = { ->
         ':!internal/venice-test-common/*',
         ':!services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java',
         ':!internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerClient.java',
+        ':!services/venice-controller/src/main/java/com/linkedin/venice/controller/init/*',
```

Comment:
To make diff coverage check pass. None of the following existing classes under the `init` package has unit test:
```
ClusterLeaderInitializationManager
ClusterLeaderInitializationRoutine
InternalRTStoreInitializationRoutine
SystemSchemaInitializationRoutine
```
I added a new routine to register meta system store schema via controller client. It's similar to SystemSchemaInitializationRoutine, tested via integ tests instead of unit tests. The diff coverage check fails as the new routine does not have unit test. This line is to exclude classes under the `init` package from diff coverage check as they do not have unit tests.